Public/Src/IDE/VsCode/BuildXL.IDE.VsCode.dsc

```diff
@@ -96,11 +101,9 @@ namespace LanguageService.Server {
                         Branding.pngFile,
                         json,
 
-                        // This contains the actual extension source as well as the
-                        // node_modules that it depends on.
                         Transformer.sealDirectory({
-                            root: d`pluginTemplate/extension`, 
-                            files: globR(d`pluginTemplate/extension`)
+                            root: extensionSourceFolder, 
+                            files: globR(extensionSourceFolder)

```

Comment:
this `globR` executes at dscript evaluation time, so it will always glob and empty folder and return 0 files